packages/react-components/src/components/RemoteVideoTile.tsx

```diff
@@ -109,12 +111,24 @@ export const _RemoteVideoTile = React.memo(
         isMuted={isMuted}
         isSpeaking={isSpeaking}
         showMuteIndicator={showMuteIndicator}
-        showLabel={props.showLabel}
         personaMinSize={props.personaMinSize}
         /* @conditional-compile-remove(one-to-n-calling) */
         /* @conditional-compile-remove(PSTN-calls) */
+        // When the remote video tile is rendered as a small tile in horizontal gallery,
+        // we hide the participants name if they are in hold/connecting states.
+        showLabel={isNarrow && isCallingOrHold(participantState) ? false : props.showLabel}
+        /* @conditional-compile-remove(one-to-n-calling) */
+        /* @conditional-compile-remove(PSTN-calls) */
         participantState={participantState}
       />
     );
   }
 );
+
+/**
+ * Checks if a participant state is calling or hold.
+ * These states match the states used to render the participantStateString in VideoTile and ParticipantItem.
+ */
+const isCallingOrHold = (participantState?: ParticipantState): boolean => {
+  return !!participantState && ['Idle', 'Connecting', 'EarlyMedia', 'Ringing', 'Hold'].includes(participantState);
```

Comment:
Do all of these states fit for this? I don't know how `idle` kicks off but are we wanting to display something in the videoGallery for it?

Reply:
So Idle is the starting state of a participant, we have logic in participant item and video tile to show this state as Connecting